speeduino/src/SPIAsEEPROM/winbondflash.cpp

```diff
@@ -46,7 +46,7 @@ typedef struct {
     winbondFlashClass::partNumber pn;
     uint16_t id;
     uint32_t bytes;
-    uint16_t pages;
+    uint32_t pages;
```

Comment:
This was a bug in the orginal code. It build fine with the arduino IDE. But not with platformio. this is the fix